pxr/base/gf/vec.template.h

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2016 Pixar
```

Comment:
Our policy has been not to update copyright years when making changes to existing files. Could I ask you to revert this bit and regenerate the code? Thanks!

Reply:
Done!